packages/spor-elm/src/Spor/LineTag/TravelTag.elm

```diff
@@ -260,3 +274,78 @@ backgroundColor variant =
 
         Walk ->
             Alias.toCss Alias.white
+
+
+deviationIcon : Maybe DeviationLevel -> Html msg
+deviationIcon maybeDeviationLevel =
+    case maybeDeviationLevel of
+        Just deviationLevel ->
+            Html.span
+                [ Attributes.css [ deviationStyle deviationLevel ] ]
+                [ Svg.fromUnstyled <|
+                    case deviationLevel of
+                        Critical ->
+                            Feedback.errorFill18X18 []
+
+                        Major ->
+                            Feedback.warningFill18X18 []
+
+                        Minor ->
+                            Feedback.warningFill18X18 []
+
+                        Info ->
+                            Feedback.informationFill18X18 []
+                ]
+
+        Nothing ->
+            Html.text ""
+
+
+deviationStyle : DeviationLevel -> Style
+deviationStyle deviationLevel =
+    let
+        fillColor =
+            if deviationLevel == Info then
+                [ Css.fill <| Alias.toCss Alias.ocean ]
+
+            else
+                []
+    in
+    Css.batch
+        [ Css.position Css.absolute
+        , Css.top <| Css.px -7
+        , Css.right <| Css.px -8
+        , Css.zIndex <| Css.int 1
+        , Css.property "paint-order" "stroke"
+        , Css.property "stroke" "white"
+        , Css.property "stroke-width" "2"
```

Comment:
For at utropstegn og kryss inne i ikonene ikke skal påvirkes av `stroke`, kan attributtet `"paint-order" "stroke"` brukes (ellers ville utropstegn og kryss blitt tykkere). Den bestemmer rekkefølgen fill, stroke og markører skal tegnes opp i.

Stroke-width settes til 2, som egentlig da tilsvarer 1 px.